Languages/IronPython/Tests/modules/network_related/_ssl_test.py

```diff
@@ -263,37 +249,37 @@ def test_SSLType_ssl():
     #TODO!
 
 
-@retry_on_failure
-def test_SSLType_ssl_neg():
-    '''
-    See comments on test_SSLType_ssl.  Basically this needs to be revisited
-    entirely (TODO) after we're more compatible with CPython.
-    '''
-    s = socket.socket(socket.AF_INET)
-    s.connect((SSL_URL, SSL_PORT))
+# @retry_on_failure
```

Comment:
Any reason why this is commented out instead of just being disabled using the `@disabled` decorator?